source/extensions/tracers/lightstep/config.h

```diff
@@ -17,9 +17,13 @@ namespace Lightstep {
 class LightstepTracerFactory : public Server::Configuration::TracerFactory {
 public:
   // TracerFactory
-  Tracing::HttpTracerPtr createHttpTracer(const Json::Object& json_config,
+  Tracing::HttpTracerPtr createHttpTracer(const envoy::config::trace::v2::Tracing& configuration,
                                           Server::Instance& server) override;
 
+  ProtobufTypes::MessagePtr createEmptyConfigProto() override {
+    return ProtobufTypes::MessagePtr{std::make_unique<envoy::config::trace::v2::LightstepConfig>()};
```

Comment:
I think you can just `return std::make_unique<...>();` without wrapping into MessagePtr